src/Core/Field.h

```diff
@@ -511,6 +510,7 @@ class Field
             case Types::IPv4:    return get<IPv4>()    < rhs.get<IPv4>();
             case Types::IPv6:    return get<IPv6>()    < rhs.get<IPv6>();
             case Types::Float64:
+                static constexpr int nan_direction_hint = 1; /// Put NaN at the end
```

Comment:
It's  because it's easy to miss (or at least I did when I introduced it) that this depends on the operation (`<` vs `>`) and that it also affects how `NULLs` are treated, so I prefer having the declaration and comment right where it's used,